### DISCUSSION TITLE
Scale is off

### DISCUSSION THREAD

**Discussion by omriyaloz** [eliemichel/MapsModelsImporter#272] [General]
Hi, everything technically working fine, but the scale is off. Imports too small. Is there a number by which I can scale the entire thing to match real world scale? 
Thanks.

**Comment by t0stiman** [comment]
Did you ever find an answer to this?

**Comment by eliemichel** [comment]
This question has been asked a lot, unfortunately after trying much I did not find a reliable way to evaluate the scaling. You need to pick 2 points, measure their real distance then scale proportionnally.
